documentation/modules/ROOT/pages/connectors/postgresql.adoc

```diff
@@ -1817,8 +1817,9 @@ It is possible to capture changes in a PostgreSQL database that is running in li
   The role grants permissions to manage logical slots and to stream data using logical slots.
   By default, only the master user account on AWS has the `rds_replication` role on Amazon RDS.
   To enable a user account other than the master account to initiate logical replication, you must grant the account the `rds_replication` role.
-  For example, `grant rds_replication to _<my_user>_`. Superuser access is required to grant the role.
-  To enable the non-master account to perform an initial snapshot, the account also requires `SELECT` permission on the schema of any tables to be captured.
+  For example, `grant rds_replication to _<my_user>_`. You must have `superuser` access to grant the `rds_replication` role to a user.
+  To enable accounts other than the master account to create an initial snapshot, you must grant `SELECT` permission to the accounts on the tables to be captured.
+  For more information about security for PostgreSQL logical replication, see the link:https://www.postgresql.org/docs/10/logical-replication-security.html[PostgreSQL documentation].
```

Comment:
Re your question on versions in PG docs elsewhere, I'd suggest to link to "latest", or whatever their alias for that is. It may result in broken links if a chapter gets removed there from one version to another, but from what I've seen, this seems rather unlikely.

Reply:
Ah, good thought. I see that URLs can specify `current` as the version. I've updated accordingly.